docs/Buttons.md

```diff
@@ -925,7 +925,8 @@ const EditToolbar = () => {
         <DeleteWithConfirmButton
             confirmContent="You will not be able to recover this record. Are you sure?"
             confirmColor="warning"
-            translateOptions={{ name: record.name }}
+            contentTranslateOptions={{ name: record.name }}
```

Comment:
I'm sorry to say but there are more places :grimacing: 

- https://github.com/marmelab/react-admin/blob/58089aa5885086bc9844064bd3512449328ff7f9/docs/Architecture.md#L250
- https://github.com/marmelab/react-admin/blob/28a6fd2c531bd6f9bf703c1772d0fd2bc24fd535/docs/Confirm.md#L74
- https://github.com/marmelab/react-admin/blob/28a6fd2c531bd6f9bf703c1772d0fd2bc24fd535/docs/Confirm.md#L77